packages/client/package.json

```diff
@@ -55,10 +55,10 @@
       "require": "./dist/links/splitLink.js",
       "default": "./dist/links/splitLink.js"
     },
-    "./links/wsLink": {
-      "import": "./dist/links/wsLink.mjs",
-      "require": "./dist/links/wsLink.js",
-      "default": "./dist/links/wsLink.js"
+    "./links/wsLink/wsLink": {
+      "import": "./dist/links/wsLink/wsLink.mjs",
+      "require": "./dist/links/wsLink/wsLink.js",
+      "default": "./dist/links/wsLink/wsLink.js"
```

Comment:
Not sure about those changes, could you help ? 

- packages/client/package.json
- packages/client/rollup.config.ts
- packages/client/src/links.ts